types/react-native/index.d.ts

```diff
@@ -1,6 +1,6 @@
 // Type definitions for react-native 0.44
 // Project: https://github.com/facebook/react-native
-// Definitions by: Eloy Durán <https://github.com/alloy>, Fedor Nezhivoi <https://github.com/gyzerok>
+// Definitions by: Eloy Durán <https://github.com/alloy>, Fedor Nezhivoi <https://github.com/gyzerok>, HuHuanming <https://github.com/huhuanming>
```

Comment:
This header isn't named the best. Adding yourself to this list means you want to be a maintainer and you will get notifications for each React Native related PR. Did you really intend to do that?